bin/development

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env ruby
 
-require File.join(File.dirname(__FILE__), '..', 'lib', 'parity')
+$LOAD_PATH.unshift File.expand_path(File.join("..", "..", "lib"), __FILE__)
+require "parity"
```

Comment:
@geoffharcourt Just did a quick test, and it works fine, while not exhaustive I think this should be okay.

The file path is always prepended to the load path. Which means it will always look for the `parity` library in that location and look else if necessary. Since the load path is prepended whenever the executable is called it will always look for "parity" in that load file first. Assuming they would load a command from the Gemfile version first with something like `bundle exec` it should get the version based on what is prepended in the line before. If it uses the executable from a package, it will never load those files in the load path to begin with since I think the load path is cleaned up after each execution.

That's a bit long winded, but I think I'm saying it's all good.

I'm also happy to do further testing.